examples/standalone/02_hello_window/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "wgpu-example-02-hello-window"
+edition = "2021"
+rust-version = "1.83"
+publish = false
+
+[dependencies]
+env_logger = "0.11.6"
+pollster = "0.4"
+wgpu = "24.0.0"
+winit = "0.30.8"
```

Comment:
In order for CI to pass, you need to specify `features = ["android-native-activity"]` and you should add a comment that you only need this if you compile to android.